pennylane_qiskit/converter.py

```diff
@@ -160,8 +170,24 @@ def _function(params: dict = None, wires: list = None):
             operation_wires = [wire_map[(qubit.register.name, qubit.index)] for qubit in op[1]]
 
             if instruction_name in inv_map and inv_map[instruction_name] in pennylane_ops.ops:
+                # Extract the bound parameters from the operation. If the bound parameters are a
+                # Qiskit ParameterExpression, then replace it with the corresponding PennyLane
+                # variable from the var_ref_map dictionary.
+
+                parameters = []
+                for p in op[0].params:
+
+                    _check_parameter_bound(p, var_ref_map)
+
+                    if isinstance(p, ParameterExpression):
+                        if p.parameters:
+                            parameters.append(var_ref_map.get(min(p.parameters)))
+                        else:
+                            parameters.append(float(p))
+                    else:
+                        parameters.append(p)
```

Comment:
Don't feel particularly strongly about this, but this could be pulled into a helper method, so `parameters` then gets constructed as a list comprehension.

Reply:
I had originally had this as a list comprehension, but changed it to a loop for readability (it became quite long due to the ternary expressions)